docs/sql/index.md

```diff
@@ -56,8 +56,375 @@ When you return data in CSV or raw format, each row corresponds to a *document*,
 
 ## User interfaces
```

Comment:
Let's mention right at the top that, y'know, we have a UI now, it's good for use cases X and Y, and include a screenshot.

Reply:
Added.

Reply:
@ashwinkumar12345 Could you help change the `GET` request above to the correspondent `POST` request? Because we're removing `GET` endpoint very soon due to security concern.

Reply:
Changed.

Reply:
Sorry for the confusion but a `POST` request should look like the sample call here: https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/sql-support.html.

Reply:
Changed.